src/prerna/semoss/web/services/local/NameServer.java

```diff
@@ -123,7 +123,7 @@ public StreamingOutput getPlaySheets(@Context HttpServletRequest request) {
 
 		List<String> sheetNames = PlaySheetRDFMapBasedEnum.getAllSheetNames();
 		for (int i = 0; i < sheetNames.size(); i++) {
-			hashTable.put(sheetNames.get(i), PlaySheetRDFMapBasedEnum.getClassFromName(WebUtility.inputSanitizer( sheetNames.get(i))));
+			hashTable.put(sheetNames.get(i), PlaySheetRDFMapBasedEnum.getClassFromName(WebUtility.inputSanitizer(sheetNames.get(i))));

```

Comment:
can we just clean sheetname instead if we need to? instead of doing this in the for loop